### PR TITLE
New design for forgot password page

### PR DESCRIPTION
**Pull request by ganeshmurali-dev**
1. Login page image issues and dashboard image issues removed.

2. Forgot password UI Design changed. 

3. Login page background image height issue fixed

### CODE REVIEW

File: src/pages/forgotPage/ForgotPage.jsx [hunk 1]
```diff
@@ -21,70 +21,76 @@ const ForgotPassword = () => {
...
+            <img src={forgotPasswordImage} alt="ForgotPasswordImage" />
           </div>
+          <div className="">
+            <div className="forgot-form-container">
+             
+                <a className="back" href="/">
+                  <span>
+                    <img src="/icons/backIcon.svg" alt="" />
+                  </span>
+                  <p className="backtologin">Back to login</p>
```
Comment (avinashdckap) [inline src/pages/forgotPage/ForgotPage.jsx:47]: There is no highlight when you highight back to login!  Is it same in design?

File: src/pages/forgotPage/ForgotPage.jsx [hunk 2]
```diff
@@ -21,70 +21,76 @@ const ForgotPassword = () => {
...
+                        onChange={handleEmail}
+                        name="email"
+                      />
+                    </div>
 
-                <p className="error-message">
-                  {emailError.email ? emailError.email : ""}
-                </p>
-              </div>
+                    <p className="error-message">
```
Comment (avinashdckap) [inline src/pages/forgotPage/ForgotPage.jsx:76]: Email  Field is required - Required field cannot be empty.
Email not valid  - Invalid Email

**Review by avinashdckap — CHANGES_REQUESTED** [review]

**Comment by avinashdckap** [comment]
On clicking on RHINO Logo redirect to Login Page

**Comment by avinashdckap** [comment]
I Feel the ICON and form is not exactly at the center for my resolution. Not sure it's same for others
Screenshot from 2024-01-05 16-37-43

File: src/pages/forgotPage/ForgotPage.jsx (hunk 1 above)
Comment (ganeshmurali-dev) [reply src/pages/forgotPage/ForgotPage.jsx:47]: Yes. Same in design

**Comment by ganeshmurali-dev** [comment]
> On clicking on RHINO Logo redirect to Login Page

Fixed this issuse

**Comment by ganeshmurali-dev** [comment]
Changed border color when clicking Inputs. Redirected when clicking logo and alignment all fixed

File: src/pages/login/Login.jsx [hunk 3]
```diff
@@ -115,7 +115,7 @@ const Login = () => {
                       name="password"
                       placeholder="Type here..."
                       {...register("password", {
-                        required: "Password is required",
+                        required: "Invalid Password",
```
Comment (avinashdckap) [inline src/pages/login/Login.jsx]: This should not be invalid password

Required field cannot be empty!  Change it

**Review by avinashdckap — CHANGES_REQUESTED** [review]

File: src/pages/login/Login.jsx (hunk 3 above)
Comment (ganeshmurali-dev) [reply src/pages/login/Login.jsx]: Changed the error message name.

File: src/pages/forgotPage/ForgotPage.jsx (hunk 2 above)
Comment (ganeshmurali-dev) [reply src/pages/forgotPage/ForgotPage.jsx:76]: Changed. Email error messages

**Pull request merged**